src/main/java/nextstep/optional/User.java

```diff
@@ -33,9 +39,22 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
+        Optional<User> optionalUser = Optional.ofNullable(user);
+
+        if (optionalUser.isPresent()) {
+            return optionalUser.map(it -> Optional.ofNullable(it.age))
+                    .filter(Optional::isPresent)
+                    .map(Optional::get)
+                    .filter(User::validateAgeRange)
+                    .isPresent();
+        }
         return false;
```

Comment:
다음과 같이 줄여서 작성해볼 수 있을 것 같습니다.
```suggestion
        return Optional.ofNullable(user)
            .map(User::getAge)
            .filter(User::validateAgeRange)
            .isPresent();
```

Reply:
앗 Optional을 제대로 이해하지 못하고 사용했던 것 같네요. 다음 커밋에는 수정해보겠습니다!